app/User.php

```diff
@@ -31,13 +31,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $fillable = [
-        'name',
-        'username',
-        'email',
-        'password',
-        'app_register_ip',
-        'email_verified_at',
-        'last_active_at'
```

Comment:
Why did you remove this field? Won't it break?

Reply:
Yes, this was a typo and is now fixed.